ChangeFeedLabSolution/ChangeFeedFunction.csproj

```diff
@@ -1,9 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net461</TargetFramework>
+    <LangVersion>preview</LangVersion>

```

Comment:
Assuming the target framework is bumped, this can be changed to `latest`.    We should try to avoid `preview` for samples.